Gemfile

```diff
@@ -5,10 +5,13 @@ gemspec
 gem 'addressable', '~> 2.6.0'
 gem 'ammeter'
 gem 'appraisal'
+gem 'capybara'
 gem 'database_cleaner', '~> 1.0'
 gem 'factory_bot_rails', '~> 5.0'
 gem 'nokogiri', '~> 1.10.0'
 gem 'pry', require: false
+gem 'rails-controller-testing'
+gem 'rspec-rails', '~> 4.0.0.beta3'
```

Comment:
In future change, we should check if 4.0.0 is good.

Reply:
Good point. I'll follow up with a PR to bump some versions.